### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_NBExtensionsTab_Inspect: Label is missing for Checkbox on page.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3993]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106
Operating System:Windows10

**Repro Steps:**
1. Login with the appropriate credentials to azure notebook account.
2. Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3. Click on Run or right click and select Run to run the notebook.
4. Navigate to NBExtensionTab and Hit enter.
5. Verify label relationship of controls on page(i.e. for Checkbox).

**Actual:**
Label is missing for checkbox form element on page.

**Expected:**
Form element or checkbox should have meaningful label on page.

**User Impact:**
Screen reader user would not get information about what type of checkbox present on page.

**Recommendation:**
Fix any of the following:
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- Element has no title attribute or the title attribute is empty

**Attachement:**
![A11y_azurenotebooks_jupyter_nbextensionstab_inspect](https://user-images.githubusercontent.com/39204909/45811832-cef06e80-bceb-11e8-9412-38d267bfc937.png)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
